pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/rbac/service_account.go

```diff
@@ -43,6 +44,11 @@ func (f *ServiceAccount) SetTemplateDefaults() error {
 const serviceAccountTemplate = `apiVersion: v1
 kind: ServiceAccount
 metadata:
+  labels:
+    app.kubernetes.io/name: controller-manager-sa
+    app.kubernetes.io/component: manager
```

Comment:
```suggestion
    app.kubernetes.io/component: rbac
```

Reply:
updated in `kustomize/v1` plugin in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/8a2c1d7c48ec7743828b14a04ae9de765cde30b2

Reply:
Should be finished in both `kustomize` plugins as of commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/daaca98d6775fe54fda2a0017d2f0f4b1369c1de

Marking this particular comment as resolved.